docs-v2/pages/connect/components.mdx

```diff
@@ -0,0 +1,501 @@
+import { Steps, Tabs } from 'nextra/components'
+import ArcadeEmbed from '@/components/ArcadeEmbed'
+import Callout from '@/components/Callout'
+import Image from 'next/image'
+
+# Running components for your end users
+
+Pipedream Connect provides you with the ability to pre-configure, deploy and
+invoke [components](/components) **on behalf of [your end
+users](/connect/api#external-users)**.
+
+## What are components?
+
+In the context of Pipedream Connect, components are self-contained executable
+units of code. They are usually provided with some arbitrary input and produce a
+result that's exported as output. These components are developed by the
+Pipedream community and their source code is available in our [public Github
+repository](https://github.com/PipedreamHQ/pipedream/tree/master/components).
+
+To learn more about components in detail, please visit our [Components
+guide](/components).
+
+## How to run components for your end users
```

Comment:
Do you see how `pages/connect/workflows.mdx` uses the `Steps` component? You should be able to wrap the headers in this section in `<Steps> ... </Steps>` and it will generate the nice numbered sections on build.